FFMpegSharp.Tests/VideoTest.cs

```diff
@@ -54,23 +89,57 @@ public void Video_ToMP4()
             Assert.IsTrue(Convert(VideoType.Mp4));
         }
 
+        [TestMethod]
+        public void Video_ToMP4_Args()
+        {
+            var container = new ArgumentsContainer();
+            container.Add(new VideoCodecArgument(VideoCodec.LibX264));
+            Assert.IsTrue(Convert(VideoType.Mp4, container));
+        }
+
         [TestMethod]
         public void Video_ToTS()
         {
             Assert.IsTrue(Convert(VideoType.Ts));
         }
 
+        [TestMethod]
+        public void Video_ToTS_Args()
+        {
+            var container = new ArgumentsContainer();
+            container.Add(new VideoCodecArgument(VideoCodec.MpegTs));
+            Assert.IsTrue(Convert(VideoType.Ts));
+        }
+
 
         [TestMethod]
         public void Video_ToOGV_Resize()
         {
-            Assert.IsTrue(Convert(VideoType.Ogv, true, VideoSize.Ed));
+            Assert.IsFalse(Convert(VideoType.Ogv, true, VideoSize.Ed));
```

Comment:
I think these tests should assert if the conversion is successful. `IsTrue` should pass for all conversion options, if the arguments are required in order for the conversion to be successful, the API will break.